test/service/actions/websocket.test.js

```diff
@@ -1,5 +1,6 @@
 // Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
 // See License.txt for license information.
+global.WebSocket = require('ws');
```

Comment:
Maybe move this line into sanity.test.js? That's where the other global setup is done in a before hook so it's ran even before any test suites